packages/front-end/pages/_app.tsx

```diff
@@ -51,6 +53,10 @@ type ModAppProps = AppProps & {
   };
 };
 
+export const stripePromise = loadStripe(
+  process.env.NEXT_PUBLIC_STRIPE_PUBLISHABLE_KEY || ""
+);
```

Comment:
Revert all changes to the `_app.tsx` file.  The logic of instantiating stripe and adding the wrapper should only be done on the specific billing components.  We don't want any stripe-related API calls to happen unless someone visits `/billing` on cloud (or in the future opens the upgrade modal on cloud).